kubernetes/operationcode_python_backend/base/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: back-end
+spec:
+  replicas: 2
+  revisionHistoryLimit: 5
+  template:
+    spec:
+      containers:
+      - name: app
+        image: operationcode/back-end:latest
```

Comment:
Do you currently have a dockerhub repo setup for this?

Reply:
Nope

Reply:
https://cloud.docker.com/repository/registry-1.docker.io/operationcode/back-end

You do now. Need to get credentials in circle ci to handle it.